### PR TITLE
Assignment2b

### PR DESCRIPTION
**Pull request by toasterdoom**

### CODE REVIEW

File: Assignment2/toasterdoom-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "@context": {
+        "student": "http://fi.upm.es/alumnos/b.agirre",
+        "subject": "http://fi.upm.es/GII/asignaturas"
```
Comment (paoespinozarias) [inline Assignment2/toasterdoom-fromJSONtoJSON-LD.jsonld:4]: Add / or # at the end of URI definitions.

File: Assignment2/toasterdoom-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    "student:@id": "b.agirre",
+    "student:@type": "Student",
```
Comment (paoespinozarias) [inline Assignment2/toasterdoom-fromJSONtoJSON-LD.jsonld:7]: "@id" and "@type" are JSON-LD tags, thus remove the prefix. 
You have to add the prefix at the id and type values. e.g. "student:bagirre"

File: Assignment2/toasterdoom-fromRDFtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "@context": {
+        "base": "http://assignment2.org/#",
```
Comment (paoespinozarias) [inline Assignment2/toasterdoom-fromRDFtoJSON-LD.jsonld:3]: you don't need to define this base prefix.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/toasterdoom-fromRDFtoJSON-LD.jsonld (hunk 3 above)
Comment (toasterdoom) [reply Assignment2/toasterdoom-fromRDFtoJSON-LD.jsonld]: Thanks for the feedback! It's fixed now

**Comment by toasterdoom** [comment]
I updated my fork and then reviewed your changes. But when I tried to do a pull request it said 600 files were deleted, something was wrong. I updated my fork to upstream somehow, but it automatically closed this pull request. I also reverted all my Assignment2 changes and my Assignment 3; thankfully I made a backup. To be honest, I don't know what I was doing.
I will now create another pull request with the Assignment2b fixes and Assignment3

**Pull request closed without merging**